### PR TITLE
add homemanger to dennis laptop and main

### PR DESCRIPTION
**Pull request by DerDennisOP**

### CODE REVIEW

**Review by FraFrieFa — APPROVED** [review]

**Comment by ahuston-0** [comment]
Lemme test on my server first and then I'll approve

**Comment by DerDennisOP** [comment]
> Lemme test on my server first and then I'll approve

It wont change anything on your server... besides the cache priority, which is for your server irrelevant

File: flake.nix [hunk 1]
```diff
@@ -233,7 +247,8 @@
                 hostname = system;
                 server = false;
                 users = [ user ];
-              } // builtins.removeAttrs (import ./users/${user}/systems/${system} { inherit inputs; }) [ "hostname" "server" "users" ]);
+                owner = user;
```
Comment (ahuston-0) [inline flake.nix:250]: alright so what does this do then because i thought it would have added a default owner?

**Comment by ahuston-0** [comment]
otherwise LGTM, just pls clarify that one bit

File: flake.nix (hunk 1 above)
Comment (DerDennisOP) [reply flake.nix:250]: its just for desktopians ...

Comment (RichieCahill) [reply flake.nix:250]: OK But what dose it do

Comment (DerDennisOP) [reply flake.nix:250]: adding default owners to desktopians

Comment (RichieCahill) [reply flake.nix:250]: so all desctop will have there root account set up like the system owner?  Is there a way to turn this off?

Comment (DerDennisOP) [reply flake.nix:250]: no currently not, feel free to add this functionality and make a pr

Comment (RichieCahill) [reply flake.nix:250]: Please consider the side effect of what you are doing in the futer this change shoul have been brought to evreyone. I dont know how to implment that yet and dont trust my self not to break stuff.

Comment (DerDennisOP) [reply flake.nix:250]: I'm the only one in desktopians with frafrifra, if other ones add their system to it they should read through the code first, a bit of documentation woudn't be bad either. Also I'm gonna review your pr, so pretty sure you wont break stuff

**Pull request merged**